src/sherpa_ai/utils.py

```diff
@@ -48,7 +48,60 @@ def load_files(files: List[str]) -> List[Document]:
             documents.extend(loader.load())
     logger.info(documents)
     return documents
+import json
+from typing import Dict, List
 
+class JsonToObject:
+    def __init__(self):
+        pass
+        
+    def __call__(self, json_data):
+        if isinstance(json_data, str):
+            data = json.loads(json_data)
+        else:
+            data = json_data
+            
+        for key, value in data.items():
+            if isinstance(value, dict):
+                value = JsonToObject()(value)
+            elif isinstance(value, list):
+                value = [JsonToObject()(item) if isinstance(item, dict) else item for item in value]
+            setattr(self, key, value)
+        return self
```

Comment:
I don't quite get this. Why do you use an additional `__call__` method instead of using the `__init__` method?